runner/cli.ts

```diff
@@ -87,10 +88,10 @@ function runSauceTunnel(env, args, output, callback) {
   });
 }
 
-function wrapCallback(output, done) {
-  return function(error) {
+function wrapCallback(output: NodeJS.WritableStream, done: (err: any) => void) {
+  return function(error: any) {
```

Comment:
My vote is `any` unless we have positive knowledge otherwise. Better broad but accurate than narrow and false.